packages/UI/src/components/Button/Button.stories.tsx

```diff
@@ -1,37 +1,62 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { within, userEvent, waitFor } from '@storybook/testing-library';
 import * as React from 'react';
 import Button from './Button';
-import { within, userEvent, waitFor } from '@storybook/testing-library';
-import { ComponentStory, ComponentMeta } from '@storybook/react';
 
 export default {
   title: 'Common Components/Button',
   component: Button,
 } as ComponentMeta<typeof Button>;
 
-export const WithShortText: ComponentStory<typeof Button> = (args) => (
-  <Button {...args} style={{ color: 'black' }}>
-    Button
-  </Button>
+export const Primary: ComponentStory<typeof Button> = (args) =>
+  // <Button {...args} as="button" disabled>
+  //   Button
+  // </Button>
+  React.createElement(Button, { disabled: true, as: 'button' }, 'Button');
+export const PrimaryLarge: ComponentStory<typeof Button> = (args) => (
+  <Button {...args}>Button</Button>
 );
 
-WithShortText.play = async ({ canvasElement }) => {
-  const canvas = within(canvasElement);
-  canvas.queryByText('Button');
-};
-
-export const WithLongText: ComponentStory<typeof Button> = (args) => (
-  <Button {...args} style={{ color: 'black' }}>
-    Welcome to Dev Launchers Platform
-  </Button>
+export const Secondary: ComponentStory<typeof Button> = (args) => (
+  <Button {...args}>Button</Button>
+);
+export const SecondaryLarge: ComponentStory<typeof Button> = (args) => (
+  <Button {...args}>Button</Button>
 );
```

Comment:
To follow the D.R.Y concept, you can see that these functions/components named `Primary` `PrimaryLarge` `Secondary` `SecondaryLarge` `Alternate` `AlternateLarge` have the same exact body definition

This can be refactored using the args template method. https://storybook.js.org/docs/react/writing-stories/introduction#using-args
I suggest reading that whole ``using-args`` part of the page